src/CMakeLists.txt

```diff
@@ -1091,7 +1091,6 @@ install(PROGRAMS
   ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/ceph-post-file
   ${CMAKE_SOURCE_DIR}/src/ceph-run
   ${CMAKE_SOURCE_DIR}/src/ceph-rest-api
-  ${CMAKE_SOURCE_DIR}/src/vstart.sh
```

Comment:
@tchaikov I always run vstart out of my build directory and I'm all for removing it, but I have run it from /usr/local/bin. Do you think a subset of ceph developers still runs it that way? @cbodley have anything to weigh in?